karmaTests/simpleKarma.ts

```diff
@@ -0,0 +1,4 @@
+const assert = require('chai').assert
+it('should equal 2', () => {
```

Comment:
can you wrap it in describe block as well?

Reply:
The initial test I used from Karma site did have a describe but then it basically said the same 0 of 0 success so I made a more simple test like this one but same result :(